bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
If we're replacing this string with a preexisting global string, should we remove this tag?